resources/todomvc/big-dom-generator/utils/buildComplex.js

```diff
@@ -2,32 +2,51 @@ const fs = require("fs");
 const { JSDOM } = require("jsdom");
 const path = require("path");
 
-const TARGET_DIRECTORY = "./dist";
-const COMPLEX_DOM_HTML_FILE = "index.html";
-const TODO_HTML_FILE = "index.html";
-const CSS_FILES_TO_ADD_LINKS_FOR = ["big-dom-generator.css"];
+function buildComplex(options) {
+    const {
+        callerDirectory,
+        sourceDirectory,
+        title,
+        filesToMove,
+        cssFilePath,
+        cssFolder = "", // sometimes the css file we are looking for may be nested in another folder.
+        cssFileNamePattern, // The css file name pattern is used to find the css file in the source dist directory.
```

Comment:
maybe write clearly in a comment that cssFileNamePattern is mandatory if cssFilePath is present? Maybe it could even be good to check this in the code and throw an intelligible error if that's absent, but I'll let you chose if you want to implement this improvement.

Reply:
Will include in the follow-up, I like the improvement.